packages/@webex/internal-plugin-calendar/src/calendar.decrypt.helper.js

```diff
@@ -36,6 +36,14 @@ const DecryptHelper = {
         )
       : [];
 
+    // Decrypt encryptedScheduleFor properties if meeting object contains SOB
+    const encryptedScheduleFor = Promise.all(
```

Comment:
question: Where is the unit test for this?

Reply:
I just add unit test for it, please review again.